Traceback (most recent call last):
  File "train.py", line 10, in <module>
    from tensorflow.contrib import learn
  File "<frozen importlib._bootstrap>", line 1007, in _handle_fromlist
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow/__init__.py", line 50, in __getattr__
    module = self._load()
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow/__init__.py", line 44, in _load
    module = _importlib.import_module(self.__name__)
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow_core/contrib/__init__.py", line 59, in <module>
    from tensorflow.contrib import labeled_tensor
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow_core/contrib/labeled_tensor/__init__.py", line 21, in <module>
    from tensorflow.contrib.labeled_tensor.python.ops import core as _core
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow_core/contrib/labeled_tensor/__init__.py", line 24, in <module>
    from tensorflow.contrib.labeled_tensor.python.ops import ops as _ops
  File "/Volumes/Transcend/opt/anaconda3/envs/keras/lib/python3.6/site-packages/tensorflow_core/contrib/labeled_tensor/python/ops/ops.py", line 1013
    name for name not in multiples
                    ^
SyntaxError: invalid syntax
